layers/generate_layers.py

```diff
@@ -466,6 +550,32 @@ def summarizeCarAvailability(row):
         "average_cars_per_household": round(total_cars / total_households, 1),
     }
 
+def generateGeojsonAndPmTiilesFromOSMFile(osmFilePath, finalOutputFilename):
```

Comment:
I am choosing to leave both files in so that the user can decide after the fact which is the better one to use, for example in the case of railway stations the geojson is still smaller than the pmtiles, probably due to being only points